src/Decompiler/Decompiler.cs

```diff
@@ -64,11 +64,10 @@ public Decompiler(ILoader ldr, IServiceProvider services)
         /// <summary>
         /// Main entry point of the decompiler. Loads, decompiles, and outputs the results.
```

Comment:
Method do not load file more. Document need to be updated.

Reply:
Fixed.